packages/gmp-ecm/gmp-ecm.7.0.3/opam

```diff
@@ -0,0 +1,52 @@
+opam-version: "2.0"
+maintainer: "7895506+MSoegtropIMC@users.noreply.github.com"
+authors: [
+  "Cyril Bouvier"
+  "David Cleaver"
+  "Pierrick Gaudry"
+  "Brian Gladman"
+  "Jim Fougeron"
+  "Laurent Fousse"
+  "Alexander Kruppa"
+  "Francois Morain"
+  "Dave Newman"
+  "Jason S. Papadopoulos"
+  "Paul Zimmermann"
+]
+homepage: "https://gitlab.inria.fr/zimmerma/ecm"
+license: ["GPL-3.0" "LGPL-3.0"]
+dev-repo: "git+https://gitlab.inria.fr/zimmerma/ecm.git"
+bug-reports: "https://gitlab.inria.fr/zimmerma/ecm/-/issues"
+build: [
+  [ "autoreconf" "-i" "-s" ]
+  [ "./configure" "--prefix" prefix
+    # Options for macOS+MacPorts
+    "CPPFLAGS=-I/opt/local/include" { os-distribution = "macports" & os = "macos" }
+    "LDFLAGS=-L/opt/local/lib" { os-distribution = "macports" & os = "macos" }
+    # Options for macOS+homebrew on Apple silicon (Intel silicon doesn't need options)
+    "CXXFLAGS=-I/opt/homebrew/include" { os-distribution = "homebrew" & os = "macos" & arch = "arm64"}
```

Comment:
Hi @MSoegtropIMC 

Here CXXFLAGS should be CPPFLAGS (as I also mentioned to @mseri at https://github.com/ocaml/opam-repository/pull/19993#issuecomment-967076046 in a related context) for ecm to find gmp.h on arm64/macOS.

Reply:
Ah yes, thanks. I always get this wrong cut and pasting from the same wrong notes ...